apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -252,6 +252,23 @@ test.describe("prefill", () => {
     });
   });
 
+  test("Persist the field values when going back and coming back to the booking form", async ({
```

Comment:
@G3root Super thanks for adding the test 🙏

I have made a small fix in it. Earlier the test was passing even without your fix because the URL had prefill query params. So, I have fixed that here.